regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -104,12 +104,12 @@ var CommentView = Backbone.View.extend({
       var href = window.APP_PREFIX + parsed.path.join('/') + '#' + parsed.hash;
       // Splice section label and context title, if present
       // TODO: Build this upstream
-      var $sectionHeader = options.$parent.find('.node:first :header, .section-title:header');
+      var $sectionHeader = options.$parent.find('.node:first :header');
```

Comment:
Just noting that we may need to add this back in the future. We don't currently allow users to comment on CFR change section headers (due to technical difficulty alone). If we were to implement that, we'd want to bring this back, I think.